hack/get_ci_vm.sh

```diff
@@ -263,7 +264,7 @@ echo -e "Note: Script can be re-used in another terminal if needed."
 echo -e "${RED}(option to delete VM presented upon exiting).${NOR}"
 # TODO: This is fairly fragile, specifically the quoting for the remote command.
 echo '#!/bin/bash' > $TMPDIR/ssh
-echo "$SSH_CMD -- -t 'cd $GOSRC && exec env \"${ENVS[*]}\" bash -il'" >> $TMPDIR/ssh
+echo "$SSH_CMD -- -t 'cd $GOSRC && exec env ${ENVS[*]} bash -il'" >> $TMPDIR/ssh
```

Comment:
This is beyond the scope of this review, but the above will fail pretty badly if any value in `ENV` has spaces. It's not worth the effort to deal with it properly, but I think a big fat warning (here and/or line 162) would be in order.

Reply:
Yep, and I've experienced it.  I'll take care of this in the beefed up comment on 162, since I need to touch that anyway.